internal/app/app.go

```diff
@@ -210,6 +228,14 @@ func loadChains(
 		return nil, nil, fmt.Errorf("failed to Init source chain provider: %w", err)
 	}
 
+	// Workaround to have more flexibility in keyring configuration
+	// Unfortunately, ChainProvider interface doesn't allow configuring keyring properly or passing existing one
+	provConcrete, ok := neutronChain.ChainProvider.(*cosmos.CosmosProvider)
+	if !ok {
+		return nil, nil, fmt.Errorf("failed to patch CosmosProvider config (type cast failed)")
```

Comment:
I'd write it like this:
```go
return nil, nil, fmt.Errorf("failed to cast neutron chain provider to *cosmos.CosmosProvider: got %T", neutronChain.ChainProvider)
```
the `%T` will print the type we've got here which will be more descriptive. how do you find it?

Reply:
Yup, good call